Disable tests, some of them need python and sometimes host-python
gets mixed with distro python leading to build failures.

Signed-off-by: Gustavo Zacarias <gustavo.zacarias@free-electrons.com>

diff -Nura glib-2.46.1.orig/gio/Makefile.am glib-2.46.1/gio/Makefile.am
--- glib-2.46.1.orig/gio/Makefile.am	2015-10-20 16:59:03.042212468 -0300
+++ glib-2.46.1/gio/Makefile.am	2015-10-20 17:32:26.763146133 -0300
@@ -233,7 +233,7 @@
 platform_deps += win32/libgiowin32.la
 endif
 
-SUBDIRS += . tests
+SUBDIRS += .
 
 if HAVE_FAM
 SUBDIRS += fam
diff -Nura glib-2.46.1.orig/glib/Makefile.am glib-2.46.1/glib/Makefile.am
--- glib-2.46.1.orig/glib/Makefile.am	2015-10-20 16:59:03.066213318 -0300
+++ glib-2.46.1/glib/Makefile.am	2015-10-20 17:33:23.537155988 -0300
@@ -33,8 +33,8 @@
 MAYBE_PCRE = pcre 
 endif
 
-SUBDIRS = libcharset $(PRINTF_SUBDIR) $(MAYBE_PCRE) update-pcre . tests
-DIST_SUBDIRS = libcharset gnulib pcre update-pcre tests
+SUBDIRS = libcharset $(PRINTF_SUBDIR) $(MAYBE_PCRE) update-pcre .
+DIST_SUBDIRS = libcharset gnulib pcre update-pcre
 
 AM_CPPFLAGS = 				\
 	$(glib_INCLUDES) 		\
diff -Nura glib-2.46.1.orig/Makefile.am glib-2.46.1/Makefile.am
--- glib-2.46.1.orig/Makefile.am	2015-10-20 16:59:03.003211087 -0300
+++ glib-2.46.1/Makefile.am	2015-10-20 17:32:20.923939420 -0300
@@ -6,7 +6,7 @@
 
 ACLOCAL_AMFLAGS = -I m4macros ${ACLOCAL_FLAGS}
 
-SUBDIRS = . m4macros glib gmodule gthread gobject gio po docs tests
+SUBDIRS = . m4macros glib gmodule gthread gobject gio po docs
 DIST_SUBDIRS = $(SUBDIRS) build
 
 bin_SCRIPTS = glib-gettextize
